packages/core/src/components/hotkeys/hotkey.tsx

```diff
@@ -50,6 +51,14 @@ export interface IHotkeyProps {
     group?: string;
 
     /**
+     * The platform type on which the application is running. This informs how keys will be named
+     * within the hotkey dialog. For example, on `PlatformType.MAC`, the hotkey dialog will refer to
+     * the "meta" key as <kbd>Cmd</kbd>, while on Windows, the hotkey dialog will refer to the
+     * "meta" key as <kbd>Ctrl</kbd>.
+     */
+    platformType?: PlatformType;
```

Comment:
@giladgray What do think of this approach? https://medium.com/@mweststrate/how-to-safely-use-react-context-b7e343eff076